client/.eslintrc.json

```diff
@@ -14,6 +14,6 @@
     "rules": {
         "react/react-in-jsx-scope": "off",
         "react/jsx-uses-react": "off",
-        "prettier/prettier": "error"
+        "prettier/prettier": "warn"
```

Comment:
I think need to swap this back to error

Reply:
Hi Jordan, Daniel and I agreed to keep this as warn about 2 months ago. We agreed that having "error" on will make it a hassle for developers to test / debug code because it won't let you compile unless you clean up the code. Either way, the github action I set up will raise a flag / failed run if there are any lint issues so I think it should be fine to keep it as "warn" for developers. If you are ok with this please let me know, if not please let me know too so I can swap it out asap. Thanks.

Reply:
Ok. As long as the Actions blocks it I am good with that!